test/runtests.jl

```diff
@@ -23,6 +27,7 @@ elseif "CPLEX" in ARGS
     @testset "test_with_cplex" begin
         include("test_with_cplex.jl")
     end
+
```

Comment:
@hdunham  do you plan to add tests here?

Reply:
yes they are just in the debugging_tests.jl file right now